### PR TITLE
Feature / Widget redesign

### PR DESCRIPTION
**Pull request by benjay10**
This plugin now features two extra buttons in the new "Insert" menu to insert a date and a dateTime value. The contextual card appears as normal to further insert a date, but with some bugfixes for the stuck `onlyDate` state. Also proper translations are provided and the plugin uses more Ember Appuniversum components.

### CODE REVIEW

**Comment by benjay10** [comment]
**Blocked by:**
In order for this plugin to work correctly, we need to wait for a review of the AuTimepicker component. Progress is potentially on its way here: https://github.com/appuniversum/ember-appuniversum/pull/252.
Also need to wait for a new release and include that in the this PR.

**Comment by Dietr** [comment]
For these compact cards I prefer not showing the card header and removing the expandable option. Makes it's function easier to discover and the card more compact. What do you think?

For example:
Screenshot 2022-05-13 at 14 54 41

**Comment by Dietr** [comment]
There seems to be an overflow issue with the .say-container__aside div. Not to be fixed here (will add a fix on ember-rdfa-editor). It cuts off the datepicker.

Screenshot 2022-05-13 at 14 53 01

File: translations/nl_BE.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    date: "Datum invoegen"
+    datetime: "Datum en tijd invoegen"
+
+  card:
+    title: "Datum"
+    insert-date: "Datum invoegen"
+    insert-datetime: "Datum en tijd invoegen"
+    label: "Selecteer datum"
+    hours: "uur"
+    minutes: "minuten"
```
Comment (Dietr) [inline translations/nl_BE.yaml]: Maybe we should use a capital for "Uur" and "Minuten"? Labels always start with capitals in our apps.

**Comment by benjay10** [comment]
@Dietr Indeed the datepicker being cut of is annoying, and I will also update the translations to have capitalisation.

For the collapsing: some people would possibly prefer less clutter on screen and would like the collapsing cards, although, without the collapsing, it is sleeker, more discoverable and you need less clicks to interact with dates. I can agree with both side and so I would choose for the design that fits in best with the rest of the editor and the desing language it is going for in the future.

**Comment by benjay10** [comment]
@Dietr If you want, you can commit the new lay-out you demonstrated in the image above if you have those changes.

**Pull request merged**